### PR TITLE
Fix ReaDDy conversion for types missing display data

### PR DESCRIPTION
**Pull request by blairlyons**
Problem
=======
ReaDDy conversion was failing if display data wasn't provided for every agent type.

Solution
========
Fixed the `ReaddyConverter` so that the default display data that is created actually gets used. Also fixed for the general case in `AgentData`.

## Type of change
- Bug fix (non-breaking change which fixes an issue)

Change summary:
---------------
* use default display data in `ReaddyConverter` for any particles without it provided
* create default display data in `AgentData`

### CODE REVIEW

**Review by ascibisz — APPROVED** [review]

File: simulariumio/data_objects/agent_data.py [hunk 1]
```diff
@@ -536,9 +536,35 @@ def display_type_for_agent(self, time_index: int, agent_index: int) -> DISPLAY_T
...
+        """
+        n_subpoints = self.n_subpoints[time_index][agent_index]
+        default_display_types = [
+            DISPLAY_TYPE.FIBER, 
+            DISPLAY_TYPE.SPHERE_GROUP, 
+            DISPLAY_TYPE.SPHERE,
+        ]
+        for display_type in default_display_types:
+            if n_subpoints % float(SUBPOINT_VALUES_PER_ITEM(display_type)) == 0:
+                return display_type
```
Comment (toloudis) [inline simulariumio/data_objects/agent_data.py:565]: this is potentially a very wrong guess in the future.  there could easily end up being multiple display types with the same number of subpoints per agent.

Comment (toloudis) [inline simulariumio/data_objects/agent_data.py:565]: there is something funky about this indirection..  are you relying on the fact that the subpoint values per item is different for all three?

Comment (blairlyons) [reply simulariumio/data_objects/agent_data.py:565]: yeah I am. I could make a constant dict or something with default to use for each values_per_item

Comment (toloudis) [reply simulariumio/data_objects/agent_data.py:565]: I agree with that, make it more explicit, or at least just a comment saying "if the number of points per type ever changes then we have to rethink this"

**Review by toloudis — APPROVED** [review]

**Pull request merged**